src/ck-ldb/LBManager.h

```diff
@@ -59,28 +59,29 @@ class CkLBArgs
     _lb_metaLbOn = 0;
     _lb_metaLbModelDir = nullptr;
   }
-  inline double& lbperiod() { return _autoLbPeriod; }
-  inline int& debug() { return _lb_debug; }
-  inline int& teamSize() { return _lb_teamSize; }
-  inline int& printSummary() { return _lb_printsumamry; }
-  inline int& lbversion() { return _lb_version; }
-  inline int& loop() { return _lb_loop; }
-  inline int& ignoreBgLoad() { return _lb_ignoreBgLoad; }
-  inline int& migObjOnly() { return _lb_migObjOnly; }
-  inline int& syncResume() { return _lb_syncResume; }
-  inline int& samePeSpeed() { return _lb_samePeSpeed; }
-  inline int& testPeSpeed() { return _lb_testPeSpeed; }
-  inline int& useCpuTime() { return _lb_useCpuTime; }
-  inline int& statsOn() { return _lb_statson; }
-  inline int& traceComm() { return _lb_traceComm; }
-  inline int& central_pe() { return _lb_central_pe; }
-  inline double& alpha() { return _lb_alpha; }
-  inline double& beta() { return _lb_beta; }
-  inline int& percentMovesAllowed() { return _lb_percentMovesAllowed; }
-  inline int& maxDistPhases() { return _lb_maxDistPhases; }
-  inline double& targetRatio() { return _lb_targetRatio; }
-  inline int& metaLbOn() { return _lb_metaLbOn; }
+  inline char*& treeLBFile() { return _lb_treeLBFile; }
+  inline double & lbperiod() { return _autoLbPeriod; }
+  inline int & debug() { return _lb_debug; }
```

Comment:
Is this (space before &) the style we use? I assume you'll be doing a clangformat on this stuff anyways, so probably not relevant.

Reply:
Yes.